exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -19,14 +19,26 @@
 import io.camunda.zeebe.protocol.record.ValueType;
 import java.io.IOException;
 import java.time.Duration;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ElasticsearchExporter implements Exporter {
 
+  /**
+   * Supported pattern for min_age property of ILM, we only support: days, hours, minutes and
+   * seconds. Everything below seconds we don't expect as useful.
+   *
+   * <p>See reference
+   * https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units
+   */
+  private static final String PATTERN_MIN_AGE_FORMAT = "^[0-9]+[dhms]$";
```

Comment:
❌ This excludes combinations of units like `3m30s`.

🔧 Perhaps there's a better way, but how about: `^([0-9]+d)?([0-9]+h)?([0-9]+m)?([0-9]+s)?$`? But you'll have to check against empty string as well.

Reply:
According to the linked docs and https://github.com/elastic/elasticsearch/blob/293d0302fb5a32d2947e57cbfe54251c6f61a1a5/libs/core/src/main/java/org/elasticsearch/core/TimeValue.java#L338-L369 this isn't allowed anyway, just a number + unit is.